src/main/java/appeng/client/gui/AEBaseGui.java

```diff
@@ -83,17 +83,17 @@
 
 public abstract class AEBaseGui extends GuiContainer
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this class is 201 which is greater than 200 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AClassCyclomaticComplexity)